SU2_CFD/src/fluid/CNEMOGas.cpp

```diff
@@ -41,17 +42,21 @@ CNEMOGas::CNEMOGas(const CConfig* config): CFluidModel(){
   hs.resize(nSpecies,0.0);            
   ws.resize(nSpecies,0.0);            
   DiffusionCoeff.resize(nSpecies,0.0);
+  Enthalpy_Formation.resize(nSpecies,0.0);
+  Ref_Temperature.resize(nSpecies,0.0);
   temperatures.resize(nEnergyEq,0.0);
   energies.resize(nEnergyEq,0.0);  
   ThermalConductivities.resize(nEnergyEq,0.0);
 
+  Kind_GasModel        = config->GetGasModel();
   Kind_TransCoeffModel = config->GetKind_TransCoeffModel();
 
-  frozen = config->GetFrozen();
-
+  frozen               = config->GetFrozen();
+  ionization           = config->GetIonization();
```

Comment:
Is the plan to pass in a Boolean value for ionization from the config versus having it stored in the TCLib?

Reply:
Im not quite sure I follow your question.

Reply:
@jtneedels it is already coming the a config input, that's why it's defined as **config->GetIonization();**